.github/workflows/fairwinds-insights.yml

```diff
@@ -0,0 +1,48 @@
+name: k8s best practices
+
+on:
+  push:
+    branches: [ master ]
+    paths-ignore:
+      - '.gitignore'
+      - 'LICENSE'
+      - '**.md'
+      - '.github/ISSUE_TEMPLATE/**'
+      - '.github/CODEOWNERS'
+      - docs/**
+  pull_request:
+    branches: [ master ]
+    paths-ignore:
+      - '.gitignore'
+      - 'LICENSE'
+      - '**.md'
+      - '.github/ISSUE_TEMPLATE/**'
+      - '.github/CODEOWNERS'
+      - '.vscode/**'
+      - docs/**
+  release:
+    types:
+      - published
+
+jobs:
+  insights:
+    if: github.event_name == 'pull_request'
+    runs-on: ubuntu-latest
+    container:
+      image: quay.io/reactiveops/ci-images:v11-alpine
+      env:
+        FAIRWINDS_TOKEN: ${{ secrets.FAIRWINDS_TOKEN }}
+    timeout-minutes: 5
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          ref: ${{ github.event.pull_request.head.ref }}
+          fetch-depth: 0
+      - name: scan repo with insights
+        run: |
+          git fetch origin master:master
+          yq e '.database.host = "host"' -i chart/values.yaml

```

Comment:
Yeah, this is not ideal.  Insights does not currently multiple values files but it is currently a feature request.  I agree this would be better solved with a second values file and I'm keeping an eye on the github issue.  Will plan to update this process when multiple values files are supported.